doc/source/conf.py

```diff
@@ -58,9 +58,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'2.0.0'
+version = u'2.0.1-rc1'
```

Comment:
Should this match our version from `setup.py`?

Reply:
Similar to AIT-Core I didnt bother updating the docs but I can address that at a later time if needed